config/initializers/assets.rb

```diff
@@ -3,10 +3,13 @@
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
 
-# Add additional assets to the asset load path
+# Add additional assets to the asset load path.
 # Rails.application.config.assets.paths << Emoji.images_path
+# Add Yarn node_modules folder to the asset load path.
+Rails.application.config.assets.paths << Rails.root.join('node_modules')
```

Comment:
Do we need this? Are we using Yarn?

Reply:
We're using it insofar as it's now installed by default, but we're not actually using it in that we have no actual dependencies that require it. I think it doesn't damage anything by having it, though, and it's now a default component and Rails guides may in the future assume we have Yarn if we're on a modern Rails version.